Xwt.XamMac/Xwt.Mac/ImageViewBackend.cs

```diff
@@ -53,16 +53,20 @@ protected override Size GetNaturalSize ()
 			NSImage img = Widget.Image;
 			return img == null ? Size.Zero : img.Size.ToXwtSize ();
 		}
+		
+		NSImage nsImage; // Needed to keep a ref, otherwise Xam.Mac might GC it and try to resurrect it
```

Comment:
Should this ViewBackend override `Dispose(bool)` and dispose the `nsImage`? I don't really know when `IWidgetBackend.Dispose` is called, maybe we don't need to care about this? Just a thought.

Reply:
Good catch! If Widget still holds a reference to the native part of NSImage, and ViewBackend doesn't dispose of Widget when it gets disposed, would disposing of the `nsImage` put it into a weird state? What if we just nulled out the `nsImage`?

Reply:
Hmm, not sure. Nulling it out is probably better than nothing in terms of spreading out GC pressure.

BTW, you can now rewrite `GetNaturalSize` to use `nsImage`.